### PR TITLE
Blacken stragglers

### PR DESCRIPTION
**Pull request by levbishop**
In #6361 we missed running black on the examples directory and on setup.py
This PR corrects that oversight and updates Makefile and tox.ini so that black and pylint are all run on the relevant files.
I think it would also be reasonable to run just black on the toplevel to avoid missing other future additions.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -22,9 +22,9 @@ commands =
 envdir = .tox/lint
 basepython = python3
 commands =
-  black --check {posargs} qiskit test tools
+  black --check {posargs} qiskit test tools examples
```
Comment (mtreinish) [inline tox.ini:25]: Should you add setup.py here too? Also what about the `black` target?

**Review by mtreinish — COMMENTED** [review]
One inline comment on the tox.ini. Also can you update the azure-pipelines.yml file too to include the extra paths.

File: tox.ini (hunk 1 above)
Comment (levbishop) [reply tox.ini]: 👍

**Comment by levbishop** [comment]
> One inline comment on the tox.ini. Also can you update the azure-pipelines.yml file too to include the extra paths.

Done. VS Code flagged a lot of complaints about the azure-pipelines.yml not being schema-compliant, but I guess since things were working before, that is spurious?

**Review by mtreinish — APPROVED** [review]

**Comment by mtreinish** [comment]
> > One inline comment on the tox.ini. Also can you update the azure-pipelines.yml file too to include the extra paths.
> 
> Done. VS Code flagged a lot of complaints about the azure-pipelines.yml not being schema-compliant, but I guess since things were working before, that is spurious?

Yeah, I'm not sure what it's using for a schema. They may point to issues or deprecations in the config file. But it all works as expected so nothing it's reporting is fatal.

**Pull request merged**